channel-sdk/src/main/java/com/zello/channel/sdk/Session.kt

```diff
@@ -467,31 +468,7 @@ class Session internal constructor(
     }
 
     private companion object {
-
-        /**
-         * Initialize the SDK.
-         * Loads the necessary native libraries.
-         * @return True if the libraries were successfully loaded.
-         */
-        private fun initialize(logger: SessionLogger?) {
-            if (!initialized) {
-                initialized = loadLib("opus", logger) && loadLib("util", logger)
-            }
-        }
-
-        private fun loadLib(name: String, logger: SessionLogger?): Boolean {
-            try {
-                System.loadLibrary("embeddable.zello.sdk.$name")
-                return true
-            } catch (t: Throwable) {
-                logger?.logError("Failed to load $name module", t)
-            }
-
-            return false
-        }
-
         private var initialized: Boolean = false
```

Comment:
It seems odd to have this variable as part of companion object, but I could be misunderstanding something. Since it gets set every time a new class instance is created, you could potentially get weird behavior as a result

Reply:
This initialized variable is really tracking whether we've loaded the native libraries, so it should be set once when any `Session` is first created and not modified again. I'll look into it and probably rename it at least.

Reply:
Gotcha, in that case you might consider an early return in `init` if `intialized` is already true. That way a second instance of this class wouldn't attempt to reload the libraries (though I'm assuming in practice there will only ever be one instance of `Session` at a time)